src/Compilers/CSharp/Portable/Symbols/Source/LocalFunctionSymbol.cs

```diff
@@ -129,6 +129,11 @@ internal void GetDeclarationDiagnostics(DiagnosticBag addTo)
             GetAttributes();
             GetReturnTypeAttributes();
 
+            if (IsExtern && _syntax.Body is null && _syntax.ExpressionBody is null)
+            {
+                GenerateExternalMethodWarnings(addTo);
```

Comment:
>GenerateExternalMethodWarnings(addTo); [](start = 16, length = 38)

I think we would want to have some similarity with ordinary methods in when we report these warnings.